index.html

```diff
@@ -290,6 +290,28 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
+    <footer>
+      <div class="form-div">
+        <div class="form-info">
+          <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+        <form action="https://formspree.io/f/form_id" method="post">
+        <div class="name-wrapper">
+          <div class="input-wrapper first-name-input">
+            <input type="text" id="first-name" placeholder="First Name" minLength="6" maxLength="30" required>
+          </div>
+          <div class="input-wrapper last-name-input">
+            <input type="text" placeholder="Last Name" minLength="6" maxLength="30" required>
+          </div>
+        </div>
+        <div class="email-wrapper">
+          <input type="email" placeholder="Your email" required>
+        </div>
+        <textarea cols="30" rows="10" placeholder="Your message here..." maxLength="500"></textarea>
+        <button type="button" class="btn-get">Get in touch</button>
+      </form>
```

Comment:
- [ ] As mentioned by the previous reviewer, your form still needs to be worked on to ensure it closely resembles the given design in Figma. Currently, on mobile screen `375px` the input elements are over-lapping the text. See the screenshot below, and look out for the red arrows. 
    - The submit `button` should have the same `width` as the button design in Figma.
Kindly stick to the design as much as possible
    - use the correct font, colors, images, text, margins, etc. Check [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

![image](https://user-images.githubusercontent.com/84156714/184205153-697b682e-8348-4c86-a62f-9dcebe4638c7.png)